src/components/calcite-switch/calcite-switch.scss

```diff
@@ -24,36 +24,36 @@
   --calcite-switch-track-border: #{$blk-160};
   --calcite-switch-handle-background: #{$blk-200};
   --calcite-switch-handle-border: #{$blk-100};
-  --calcite-switch-hover-handle-border: #{$ui-blue-hover-dark};
+  --calcite-switch-hover-handle-border: var(--calcite-ui-blue-hover-dark);
```

Comment:
This is what I'm seeing for dark theme - no color for red / blue is being applied for hover or active: 
![Screen Shot 2020-01-21 at 4 28 30 PM](https://user-images.githubusercontent.com/4733155/72855080-31080d80-3c6b-11ea-843e-08c1b7322aff.png)

Reply:
I fixed this one too, take another look.